### DISCUSSION TITLE
Suddenly can't upload files

### DISCUSSION THREAD

**Discussion by ekulxam** [community/community#111265] [New to GitHub] [answered]
<img width="543" alt="commit failed" src="https://github.com/community/community/assets/159592458/9e1fdcb9-d670-4bcb-bd72-84ed85337965">

Cannot upload files because I keep getting the above error

I use GitHub by directly uploading files

It worked fine yesterday

**Comment by GrabbitGames** [comment]
Same

**Comment by GrabbitGames** [comment]
It may be down

**Comment by sanmirgabriel** [comment]
same

**Comment by GrabbitGames** [comment]
What do we do! It's still not working

**Reply by Gubna-Tech** [reply]
I removed my Github pages and after that uploading worked for me again.

**Reply by ekulxam** [reply]
I don't have GitHub pages active.

**Comment by GrabbitGames** [comment, marked as answer]
It’s working now you can upload files!

**Reply by ekulxam** [reply]
Oh yeah, it just worked for me.
I'll close this discussion but reopen it if it ever happens again.

**Reply by GrabbitGames** [reply]
Alr, Have a good day!

**Reply by AliElBatrawyAli1** [reply]
i can't upload my file please solve this problem
